Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -910,13 +913,16 @@ def audit_revision_and_version_scheme
           newest_committed_version ||= previous_version
           newest_committed_checksum ||= previous_checksum
           newest_committed_revision ||= previous_revision
+          newest_committed_url ||= stable.url
         end
 
         break if previous_version && current_version != previous_version
       end
 
       if current_version == previous_version &&
```

Comment:
```suggestion
      if current_version == newest_committed_version &&
```

Another change that should be made here (unrelated). `current_version` and `previous_version` are always different so this will never be triggered anyway. Using `newest_committed_version` is correct because we want to make see if the version has changed since the last commit rather than the last version.

This will resolve #9083

Reply:
Actually, that's incorrect. `current_version` and `previous_version` _can_ be the same but only if the `current_version` is also the first version (which is why the audit triggered for `vint` but not for `nim`

To avoid this, maybe we should set `previous_version`, `previous_version_scheme` and `previous_revision` to `nil` if there isn't a previous version.